build.gradle

```diff
@@ -394,10 +394,7 @@ task testIntegration(type: Test) {
     testLogging {
         events = ["passed", "failed", "skipped", "started", "standard_out"]
     }
-    retry {
-        maxRetries = 3
-        maxFailures = 15
-    }
+    failFast = true
```

Comment:
Any concerns about some of our flaky tests?

Reply:
are flaky tests even more likely to cause this issue?

Reply:
Yes, I think flaky tests are a concern. But, even a flaky tests can cause a cascading billing profile issue, so I'd argue that  even with flaky tests, we should "fail fast", identify the flaky test and try to fix or ignore until our longer term solution is in.